### ISSUE TITLE
[feature-request] Show the number of direct children when collapsed

### ISSUE THREAD

**Issue by balta2ar** [PaulJuliusMartinez/jless#85]
e.g. keys of an object or number of elements in an array:
```
{
  "eventRecords": {…} (48),
  "registrationRecords": {…} (20344)
}
```
(also requested here: https://github.com/antonmedv/fx/issues/138)

**Comment by brewingcode** [comment]
I have this in #40, but I haven't seen @PaulJuliusMartinez around since early March, so that PR remains unmerged.

**Comment by PaulJuliusMartinez** [comment]
This has been implemented in [`v0.9.0`](https://github.com/PaulJuliusMartinez/jless/releases/tag/v0.9.0).

I'm not totally thrilled with it; it's definitely useful for arrays, but I find it to be less so for objects.
